### DISCUSSION TITLE
PDFDelegateFailed `The system cannot find the file specified. ' @ error/pdf.c/ReadPDFImage/794 http://localhost:8000

### DISCUSSION THREAD

**Discussion by jenishdonga** [spatie/pdf-to-image#190] [Q&A]
I have install spatie/pdf-to-image in laravel xamp windows and working fine  simply open http://localhost/project/convert-pdf but when in open http://localhost:8000/convert-pdf thowing PDFDelegateFailed `The system cannot find the file specified. ' @ error/pdf.c/ReadPDFImage/794

**Comment by freekmurze** [comment]
I don't have access to Windows, but I'm open to a PR that would fix the problem for you.
